config/software/expat.rb

```diff
@@ -1,21 +1,58 @@
+#
+# Copyright 2014 Chef Software, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
 name "expat"
-default_version "2.1.1"
+default_version "2.5.0"
+
+relative_path "expat-#{version}"
+dependency "config_guess"
```

Comment:
`config_guess` is `GPL-3.0 (with exceptions)`, so to be safe, we should add `ship_source_offer true` to its recipe.

It also looks like our `config_guess` definition points to [TODO](https://github.com/chef-boneyard/config-mirror) (last updated 6 years ago) instead of http://git.savannah.gnu.org/cgit/config.git/ - is this intended?

Reply:
Sounds good, I'll approve this PR then :+1: